python/dgl/nn/pytorch/sparse_emb.py

```diff
@@ -376,6 +376,36 @@ def all_set_embedding(self, values):
         if th.distributed.is_initialized():
             th.distributed.barrier()
 
+    def _all_get_tensor(self, shared_name, tensor, shape):
+        """A helper function to get model-parallel tensors."""
+        # create a shared memory tensor
+        if self._rank == 0:
```

Comment:
If the embedding is created in shared memory, will it has a legal `rank` number?